src/Access.hs

```diff
@@ -190,6 +190,7 @@ isStudent u = isJust $ find (has _Student) (u ^.. userRoles)
 
 delegationCapabilities :: User -> Maybe User -> Maybe IdeaSpace -> Maybe Phase -> [Capability]
 -- guards: no delegation in these situations
+delegationCapabilities _ _ _ (Just PhaseVoting{})              = []
```

Comment:
nope.  "never *after* voting phase" does not imply "never *in* voting phase".  we explicitly want to allow students to delegate during voting phase.

i'm not sure this issue was ever reproduced, or whether it was just a reminder to check we got it right.  i think we may have run into an independent (and already fixed) problem with the `CanDelegate*` confusion during the presentation.